physical/cockroachdb/cockroachdb_test.go

```diff
@@ -12,8 +12,6 @@ import (
 	"github.com/hashicorp/vault/helper/testhelpers/docker"
 	"github.com/hashicorp/vault/sdk/helper/logging"
 	"github.com/hashicorp/vault/sdk/physical"
-
-	_ "github.com/lib/pq"
```

Comment:
Do we need to import `github.com/jackc/pgx/v4/stdlib`?

Reply:
We don't import it in `physical/cockroachdb/cockroachdb_test.go`

Reply:
If you're referring to the import on the non-test file for this package, this is a pattern that is specific to the `database/sql` package from stdlib. By importing this way, it gets to call the driver's `init()` function which is where the package registers the driver. All of [the packages](https://github.com/golang/go/wiki/SQLDrivers), including pgx, that supports usage through `database/sql` are usually imported this way. [This blog post](https://www.calhoun.io/why-we-import-sql-drivers-with-the-blank-identifier/) provides a good insight into this topic as well.

Reply:
I know about the `database/sql` pattern. I was mostly wondering why we no longer needed it in this file after switching drivers.

Reply:
My _guess_ is that it was probably unnecessary before with pq. The code file does an unnamed import of the driver (pq, now pgx) which will register it, and since nothing here is referring to pq.* or pgx.* it probably isn't needed.